contrib/libdav1d/A00-add-cross-file.patch

```diff
@@ -0,0 +1,19 @@
+diff -Naur /dev/null dav1d-f1b756ef5bdc0bb759b2b140f15362fec024c1ff/x86_64-w64-mingw32.meson
```

Comment:
Been submitted upstream?

Reply:
Never mind, see you resolved it.

Reply:
Not yet, but happy to. @EwoutH is there a reason why I shouldn't submit this as a PR upstream to dav1d?

Reply:
Upstream merge request: https://code.videolan.org/videolan/dav1d/merge_requests/791